benchmarks/dynamo/pr_time_benchmarks/check_results.py

```diff
@@ -92,34 +99,50 @@ def log(event_name):
                 ),
             )
 
+        ratio = float(result - entry.expected_value) * 100 / entry.expected_value
+
         if result > high:
+            new_entry = copy.deepcopy(entry)
+            new_entry.expected_value = result
+            new_expected[key] = new_entry
+
             fail = True
-            ratio = float(result - entry.expected_value) * 100 / entry.expected_value
             print(
                 f"REGRESSION: benchmark {key} failed, actual result {result} "
-                f"is {ratio:.2f}% higher than expected {entry.expected_value} ±{entry.noise_margin*100:.2f}% "
-                f"if this is an expected regression, please update the expected results."
+                f"is {ratio:.2f}% higher than expected {entry.expected_value} ±{entry.noise_margin*100:+.2f}% "
+                f"if this is an expected regression, please update the expected results.\n"
```

Comment:
I know it's listed below, but would be nice if there were instructions on how to update in the msg, like with other pytorch failures that tell you to run `EXPECTTEST=1 {test}`. The ci csv is not easy to understand where to get or regenerate